src/commons/api/rrdp.rs

```diff
@@ -346,7 +345,8 @@ impl Notification {
                 {
                     // snapshot ref
                     let uri = self.snapshot.uri.to_string();
-                    let a = [("uri", uri.as_str()), ("hash", self.snapshot.hash.as_ref())];
+                    let hash = self.snapshot.hash.to_string();
```

Comment:
Is this `to_string()` operation costly compared to the prior `.as_ref()` operation? Does it matter?